test/tests/providers/global/platform/atlas/alerting-rules/logging-pipeline.rules.test.yml

```diff
@@ -0,0 +1,229 @@
+---
+rule_files:
+  - logging-pipeline.rules.yml
+
+tests:
+  # Test LoggingAgentDown
```

Comment:
This test should probably be in `alloy.rules.test.yml`, as the alert is defined in `alloy.rules.yml`

Reply:
done :)